src/components/Uploader/index.js

```diff
@@ -2,20 +2,24 @@ import { h } from 'preact'
 import Dropzone from 'react-dropzone'
 import theme from '../Theme/style.css'
 import style from './style.css'
+import { isDesktop } from '../utils'
 import {errors} from '../strings/errors'
 import { trackComponentAndMode } from '../../Tracker'
 import SwitchDevice from '../crossDevice/SwitchDevice'
 import { mobileCopy, desktopCopy } from '../strings/uploadCopy'
 
-const instructionsCopy = (method, side, allowCrossDeviceFlow) => {
-  const instructions = allowCrossDeviceFlow ? desktopCopy.instructions : mobileCopy.instructions
+const instructionsCopy = (method, side) => {
+  const instructions = isDesktop ? desktopCopy.instructions : mobileCopy.instructions
```

Comment:
This will show the component on the mobile flow if it's opened on desktop

Reply:
I think this is OK - on mobile we always want to say 'Take a photo...' and on desktop we always want to say 'Upload...'. I don't think it matters if we are on cross device or not.